src/features/Product/components/Inquiry.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import Textarea from "components/Textarea";
+import { INQUIRY_PLACEHOLDER } from "constants/message";
+import styles from "./Inquiry.module.css";
+import Button from "components/Button";
+
+const Inquiry = () => {
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  const handleTextareaChange = (event) => {
+    if (event.target.value !== "") {
+      setIsDisabled(false);
+      return;
+    }
+
+    setIsDisabled(true);
```

Comment:
P3:
아래 코드도 동일동작합니다.

```suggestion
      setIsDisabled(!!event.target.value);
```

`!` 연산자를 연속으로 두번 사용해 가독성이 떨어지시는 것 같다면 아래의 경우도 가능합니다.

```suggestion
      const isEmpty = !event.target.value;
      
      setIsDisabled(!isEmpty);
```